### DISCUSSION TITLE
Feature request: Moving text or creating new text sections

### DISCUSSION THREAD

**Discussion by SignalUser2022** [gsantner/markor#1742] [Ideas] [answered]
Dear Developer,

first of all, thank you very much for creating and maintaining this app. I've used it many times and it's very helpful, expecially because it's available on F-Droid and Open Source.
There's one feature I'm missing. Is it possible to add the option to either move text sections freely around within a meme template or create different text sections than just at the top and bottom?

That would be amazing.

Thank you :-)

**Comment by gsantner** [comment, marked as answer]
MemeTastic is one of my projects - but this is Markor. Please check issues of the MemeTastic project. :-)

**Comment by harshad1** [comment]
Within Markor we have actions to move text 1 line up or down and the ability to insert snippets. These may help.
